servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -250,10 +286,64 @@ void SceneShaderForwardClustered::ShaderData::set_code(const String &p_code) {
 
 	if (depth_test != DEPTH_TEST_DISABLED) {
 		depth_stencil_state.enable_depth_test = true;
-		depth_stencil_state.depth_compare_operator = RD::COMPARE_OP_LESS_OR_EQUAL;
 		depth_stencil_state.enable_depth_write = depth_draw != DEPTH_DRAW_DISABLED ? true : false;
+
+		RD::CompareOperator depth_function_rd_table[DEPTH_FUNCTION_MAX] = {
```

Comment:
```suggestion
		static const RD::CompareOperator depth_function_rd_table[DEPTH_FUNCTION_MAX] = {
```
Here as well (even though it's not done in the other cases it's cleaner IMO)